### PR TITLE
Add flutter_lints package

### PR DESCRIPTION
**Pull request by goderbauer**
https://github.com/flutter/flutter/issues/78432

### CODE REVIEW

File: packages/flutter_lints/lib/flutter.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Recomended lints for Flutter apps, packages, and plugins.
```
Comment (pq) [inline packages/flutter_lints/lib/flutter.yaml:1]: => Recommended

File: packages/flutter_lints/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: flutter_lints
+description: Recommended lints for Flutter apps, packages, and plugins
```
Comment (pq) [inline packages/flutter_lints/pubspec.yaml:2]: End with a period?

=> 

    Recommended lints for Flutter apps, packages, and plugins.

File: packages/flutter_lints/pubspec.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+version: 0.1.0
+homepage: https://github.com/flutter/packages/tree/master/packages/flutter_lints
+publish_to: none
```
Comment (pq) [inline packages/flutter_lints/pubspec.yaml:5]: I think you want to remove this?

Comment (goderbauer) [reply packages/flutter_lints/pubspec.yaml:5]: Ultimately, yes. However, while I pull in the "lints" dependency directly from github below, there's a lint telling me that I need this. 😝

Comment (pq) [reply packages/flutter_lints/pubspec.yaml]: ha!

**Review by devoncarew — APPROVED** [review]
👍

**Review by pq — APPROVED** [review]
💎

File: packages/flutter_lints/pubspec.yaml [hunk 4]
```diff
@@ -1,13 +1,11 @@
...
 homepage: https://github.com/flutter/packages/tree/master/packages/flutter_lints
-publish_to: none # TODO(goderbauer): remove this
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  lints: # TODO(goderbauer): depend on released version before releasing the package
-    git: https://github.com/dart-lang/lints.git
+  lints: ^1.0.0-dev.0
```
Comment (pq) [inline packages/flutter_lints/pubspec.yaml:10]: 🎉

**Pull request merged**